fyde-app/README.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Fyde App
+has_children: false
+nav_order: 3
+permalink: /fyde-app
+---
+# Fyde App
+{: .no_toc }
+
+The Fyde app bundles the end user UI and the Fyde agent. It is available on the following platforms:
+
+* iOS - available on the [App Store](https://www.fyde.com/downloads)
+* MacOS - available on the [App Store](https://www.fyde.com/downloads)
+* Windows - available on the [Microsoft Store](https://www.fyde.com/downloads)
+* Linux - repos soon available for Debian and Arch.
+
+## Fyde Agent
```

Comment:
@luisalima perhaps you can update the glossary, namely https://fyde.github.io/docs/glossary/#fyde-app to be in sync with here